src/System.Binary.Base64/System/Binary/Base64.cs

```diff
@@ -270,10 +353,12 @@ private static int DecodePadByTwo(ref byte srcBytes)
         /// <returns>Number of bytes written to the destination.</returns>
         public static int Decode(ReadOnlySpan<byte> source, Span<byte> destination)
         {
-            /*if (destination.Length < ComputeDecodedLength(source.Length))
+            int requiredLength = ComputeDecodedLength(source);
```

Comment:
Yes, of course. I was waiting till we get agreement on API surface and behavior before changing to the Try* API that would support multiple segments.